sscanss/core/util/misc.py

```diff
@@ -97,11 +97,11 @@ class LoadVector(Enum):
 
 
 @unique
-class MessageSeverity(Enum):
-    """Severity of the message box information"""
+class MessageType(Enum):
+    """Type of displayed message: information, warning or error"""
```

Comment:
its okay to just say ""Type of displayed message", listing the enum in not necessary